crates/core/tedge_agent/src/operation_workflows/persist.rs

```diff
@@ -410,14 +402,14 @@ impl WorkflowRepository {
         operation: &OperationType,
         command_state: GenericCommandState,
     ) -> Result<Option<GenericCommandState>, WorkflowExecutionError> {
+        let operation_name = operation.to_string();
```

Comment:
Minor: I think `to_owned` better signifies the intent here
```suggestion
        let operation_name = operation.to_owned();
```

Reply:
`to_owned()` is not appropriate here.

To convey the intent, I replaced `to_string()` by `name()`: https://github.com/thin-edge/thin-edge.io/pull/3309/commits/33268b91f4e499a9bb1cc2b9c79d7886680f270e